js/views/pageNavVw.js

```diff
@@ -109,119 +86,6 @@ module.exports = Backbone.View.extend({
     }
   },
 
-  refreshProfile: function() {
```

Comment:
You can't remove this, there are other places in the code that call it, such as when you finish the store wizard. The listener that fires refreshProfile is still here, so just undelete the refreshProfile function.